flipperdevices/flipperzero-firmware
# NFC: fix creating MF Classic tags from "Add Manually" menu (BCC calulation and ATQA/SAK writing)

# What's new

- Write necessary data to tag (BCC/ATQA/SAK) when creating tag from menu "Add Manually" (BCC only for 4byte tags). Fixes https://github.com/flipperdevices/flipperzero-firmware/issues/2341

# Verification 

- Create "1k 4byte" tag from menu
- Write it to magic tag
- Check that tag is readable

# Checklist (For Reviewer)
